src/redux/books/booksSlice.js

```diff
@@ -1,7 +1,24 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  bookItems: [],
+  bookItems: [{
+    id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  }],
 };
 
```

Comment:
- [ ] You did a great job making the addition and removal of books work. :+1: According to the requirements, we are supposed to input `title` and `author` attributes of the book. Kindly add the input field for the author and please also ensure to display the author when rendering the books. ✍️

**Requirements**
![image](https://github.com/katfogy/bookstore-react/assets/17243325/8a454c42-00f0-4bec-b862-2b2f44c37a05)